src/js/utils/ServiceSpecUtil.js

```diff
@@ -0,0 +1,81 @@
+import PodSpec from '../structs/PodSpec';
+import ApplicationSpec from '../structs/ApplicationSpec';
+import FrameworkSpec from '../structs/FrameworkSpec';
+import Util from './Util';
+
+var ServiceSpecUtil = {
+
+  /**
+   * Creates a new PodSpec with the scaling specifications
+   * changed to fixed, and with the number of instances defined
+   * from the second argument.
+   *
+   * @param {PodSpec} spec - The spec to change
+   * @param {number} instancesCount - The new number of instances
+   * @returns {PodSpec} - Returns a new PodSpec instance
+   */
+  setPodFixedScaling(spec, instancesCount) {
```

Comment:
Can we rename this to `setPodInstances`, so that all those methods follow the same naming scheme?  #nit